pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -274,6 +275,8 @@ public void handle(Request request, Response response) throws Exception {
             }
         });
 
+        before(API.BASE + "/*", new LanguageResolutionFilter());
```

Comment:
Should we really mount the filter for all routes or limit it to ones under `/user/{userGuid}/studies/{studyGuid}`? I mean, does it make sense outside the study context given that we use the notion "languages supported by the study"?

Reply:
That's a good point. I feel making it a filter just for `/user/{}/studies{}` routes and renaming it to something like `StudyLanguageResolveFilter` seems reasonable.

Reply:
Agreed!  If/when we move  other  translated assets (thinking of `en.json`-type things from angular), we can adjust later on.

Reply:
Done, renamed the filter and made it applicable to `/user/*studies/*` only. Additionally, put back the warning and adjusted it to suggest remounting the filter under `/user/*studies/*` if someone inadvertently enables it for other routes.